.github/workflows/sanity.yaml

```diff
@@ -11,9 +11,6 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-      - name: Install nfs-common
```

Comment:
we don't need this change?

Reply:
This change was required so that the sanity test prow job could reuse this functionality